developer-docs-site/docs/concepts/delegation_staking.md

```diff
@@ -0,0 +1,125 @@
+---
+title: "Delegation Staking"
+slug: " Delegation staking"
+---
+import ThemedImage from '@theme/ThemedImage';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+# Delegation Staking
```

Comment:
Is this delegation staking or delegated staking?

Reply:
there's no functionality to upgrade from staking to delegated staking yet.

I think either delegation staking or delegated staking works as long as we stay consistent with the naming. but maybe delegation staking would be more consistent with "delegation pool operations"

Reply:
Let's go with "delegated staking" as it sounds a bit more natural and slightly more hits on google? Plus this was the second most popular hit: https://www.coindesk.com/tech/2023/04/21/aptos-introduces-delegated-staking-to-increase-participation-in-staking-on-network/

Reply:
> there's no functionality to upgrade from staking to delegated staking yet.

Might want to call that out -- that the only way is to convert by removing and moving to a new contract there is no upgrade mechanism.

Reply:
yeah agree with david's comments

Reply:
it's updated